### PR TITLE
Speed up tests

### PR DESCRIPTION
**Pull request by jrhite**
turn off type-checking while running tests to speed them up

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +33,9 @@
     "deploy:kovan": "npx hardhat --config hardhat.kovan-staging-config.ts run scripts/kovan-deployment.ts --network kovan",
     "frontend": "cd frontend && npm run start",
     "node": "npx hardhat node --watch",
-    "test": "npx hardhat test ./test/unit/**/*.ts",
-    "test-integration": "npx hardhat --config hardhat.integration-test-config.ts test ./test/integration/**/*.ts"
+    "test": "npm run test:unit && npm run test:integration",
+    "test:unit": "TS_NODE_TRANSPILE_ONLY=1 npx hardhat test ./test/unit/*.ts",
```
Comment (Pongch) [inline package.json:37]: is there an explaination to what `TS_NODE_TRANSPILE_ONLY=1` does? curious.

**Review by Pongch — COMMENTED** [review]
It seems like the PR is building on nich's old branch. I'm not sure if this will cause merge conflict with Nich's current work given a lot of diffs in here (although the only changes here should be in the package.json). Shall we wait until Nich landed her new UI Update PR and then rebase?

File: package.json (hunk 1 above)
Comment (jrhite) [reply package.json:37]: yep, when we run our tests, typescript typechecks everything while running the tests which is very expensive. This flags turns of typechecking off, and simply transpiles all the code from TS to JS when running tests.

see here: https://hardhat.org/guides/typescript.html#performance-optimizations

**Comment by jrhite** [comment]
> It seems like the PR is building on nich's old branch. I'm not sure if this will cause merge conflict with Nich's current work given a lot of diffs in here (although the only changes here should be in the package.json). Shall we wait until Nich landed her new UI Update PR and then rebase?

@Pongch sure not urgent and can wait. actually this PR is a single file 2 line change. i can rebase it.

**Comment by jrhite** [comment]
@Pongch rebased to head of master

**Pull request merged**